actions/overlay_action.go

```diff
@@ -1,3 +1,26 @@
+/*
+Overlay Action
+
+Recursive copy of directory or file to target root filesystem.
+
+Yaml syntax:
+ - action: overlay
+   origin: name
+   source: directory
+   destination: directory
+
+Mandatory properties:
+
+- origin -- reference to named file or directory.
```

Comment:
origin isn't  mandatory. Origin defaults to the recipes directory.

Source is mandatory (unless origin refers to an unpacked download? has that been tested)

Reply:
hm... #23
Moved source to mandatory and origin to optional.